.gitignore

```diff
@@ -45,3 +45,5 @@ examples/**/terraform.zip
 # never upload the build to git
 terraform-provider-azurerm
 tf.zip
+
+azurerm/internal/services/**/validate/*.go
```

Comment:
Is this something that we need added to our .gitignore file?

Reply:
Currently, if one is using `make build` a lot of **validate/*.go files are generated, which seems that they weren't checken in. I don't know if this should be added to the `.gitignore` file. But at least all generated files should be then checked in. In my case over 270 files were generated, which was really annoying while dealing with git.

Reply:
Hmmm, I'm wondering if there is something else going on on your side. Pulling your branch down and running make build doesn't generate any files. We will need to remove this line as we'll need to have git manage our validate files.

Reply:
Done. I've to check what wrong with my setup. Thx.